addons/mail/models/mail_thread.py

```diff
@@ -1413,8 +1413,8 @@ def _message_parse_extract_payload(self, message, save_original=False):
                     continue  # skip container
 
                 filename = part.get_filename()  # I may not properly handle all charsets
-                if part.get_content_type() == 'text/xml' and not part.get_param('charset'):
-                    # for text/xml with omitted charset, the charset is assumed to be ASCII by the `email` module
+                if 'text/' in part.get_content_type() and not part.get_param('charset'):
```

Comment:
reading it a second time, let's write that instead, for consistency with other places where we did some mimetype checking (e.g. inside http.py)
```suggestion
                if part.get_content_type().startswith('text/') and not part.get_param('charset'):
```
no need to change 15.0, let's just change it now and onward